### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by luigibk**
https://tools.hmcts.net/jira/browse/RPE-932

### CODE REVIEW

File: blobstorage/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: A Helm chart to access Azure blob storage
+name: blobstorage
+version: 0.0.1
+icon: https://github.com/hmcts/chart-blobstorage/raw/master/images/message_icon-32.png
```
Comment (timja) [inline blobstorage/Chart.yaml]: image looks more relevant to the servicebus then blob storage

File: blobstorage/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+setup:
+  containers:
+   - first-container
+   - second-container
+```
+**NOTE**: at least one container and the resource group are required for the blob storage service to provision the account and container(s) required for the application.
+
+## Using it in your helm chart.
+To get the container(s) access key and blob service endpoint needed in your application you need to use the secrets map that is available once the storage account and container(s) are provisioned.
+To access the connection string secret for the desired queue we can use this snippet in your values.template.yaml:
```
Comment (timja) [inline blobstorage/README.md]:
```suggestion
To access the connection string secret for the desired container we can use this snippet in your values.template.yaml:
```

Comment (luigibk) [reply blobstorage/README.md]: Ok, removed extra line

File: blobstorage/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      key: primaryBlobServiceEndPoint
+```
+
+## Configuration
+
+The following table lists the configurable parameters of the Blob Storage chart and their default values.
+
+| Parameter      | Type | Description | Default |
+| -------------- | ---- | ----------- | ------- |
+| `location` | string |location of the PaaS instance of the blob storage to use | `uksouth` |
```
Comment (timja) [inline blobstorage/README.md]:
```suggestion
| `location` | string | location of the PaaS instance of the blob storage to use | `uksouth` |
```

File: blobstorage/README.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+| `resourceGroup` | string | This is the resource group required for the azure deployment |  **Required** |
+| `setup` | array |see the full description of the setup objects in [setup objects](#setupobjects)| **Required** |
```
Comment (timja) [inline blobstorage/README.md]:
```suggestion
| `setup` | array | see the full description of the setup objects in [setup objects](#setupobjects)| **Required** |
```

File: blobstorage/README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+| `setup.enableNonHttpsTraffic` | `string` |  Specify whether non-https traffic is enabled. | `disabled`|
+
+
+## Setup Objects
+Currently we support only multiple `container(s)` setup within a single blob storage account. This should be flexible enough to cover all use cases.
+
+ The container object definition is:
+```yaml
+setup:
+  container:
```
Comment (timja) [inline blobstorage/README.md]:
```suggestion
  containers:
```

File: blobstorage/README.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+## Development and Testing
+
+Default configuration (e.g. default image and ingress host) is setup for sandbox. This is suitable for local development and testing.
+
+- Ensure you have logged in with `az cli` and are using `sandbox` subscription (use `az account show` to display the current one).
+- For local development see the `Makefile` for available targets.
+- To execute an end-to-end build, deploy and test run `make`.
+- to clean up deployed releases, charts, test pods and local charts, run `make clean`
+
+`helm test` will deploy a busybox container alongside the release which performs a simple HTTP request against the service health endpoint. If it doesn't return `HTTP 200` the test will fail. **NOTE:** it does NOT run with `--cleanup` so the test pod will be available for inspection.
```
Comment (timja) [inline blobstorage/README.md]: is this accurate?

File: blobstorage/templates/_helpers.tpl [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{{/*
+All the common lables needed for the lables sections of the definitions.
```
Comment (timja) [inline blobstorage/templates/_helpers.tpl]:
```suggestion
All the common labels needed for the labels sections of the definitions.
```

File: blobstorage/values.yaml [hunk 8]
```diff
@@ -0,0 +1,6 @@
+location: uksouth
+resourceGroup: someGroup
+setup:
+  containers:
+   - first-container
```
Comment (timja) [inline blobstorage/values.yaml]: probably should just default to one container, 
maybe 
```suggestion
   - container
```

File: ci-values.yaml [hunk 9]
```diff
@@ -0,0 +1,5 @@
+
+environment:
```
Comment (timja) [inline ci-values.yaml]: are these relevant?

**Pull request merged**